bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,676 @@
+				<option value="LaundryCare.Washer.EnumType.SpinSpeed.UlHigh">High (US/CA)</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="laundry_care_washer_idos1">
+		<item-type>String</item-type>
+		<label>i-Dos 1 dosing level</label>
+		<description>This status defines the i-Dos dosing level of a washer program of the home appliance. (If appliance
+			supports i-Dos)</description>
+		<state>
+			<options>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Off">Off</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Light">Light</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Normal">Normal</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Strong">Strong</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="laundry_care_washer_idos2">
+		<item-type>String</item-type>
+		<label>i-Dos 2 dosing level</label>
+		<description>This status defines the i-Dos dosing level of a washer program of the home appliance. (If appliance
+			supports i-Dos)</description>
+		<state>
+			<options>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Off">Off</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Light">Light</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Normal">Normal</option>
+				<option value="LaundryCare.Washer.EnumType.IDosingLevel.Strong">Strong</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="setpoint_temperature_refrigerator">
+		<item-type>Number:Temperature</item-type>
+		<label>Refrigerator temperature</label>
+		<description>Target temperature of the refrigerator compartment (Range depends on appliance - common range 2 to 8°C).</description>
+		<state step="1" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="setpoint_temperature_freezer">
+		<item-type>Number:Temperature</item-type>
+		<label>Freezer temperature</label>
+		<description>Target temperature of the freezer compartment (Range depends on appliance - common range -16 to -24°C).</description>
+		<state step="1" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="super_mode_refrigerator">
+		<item-type>Switch</item-type>
+		<label>Refrigerator super mode</label>
+		<description>The setting has no impact on setpoint temperatures but will make the fridge compartment cool to the
+			lowest possible temperature until it is disabled manually by the customer or by the HA because of a timeout.</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="super_mode_freezer">
+		<item-type>Switch</item-type>
+		<label>Freezer super mode</label>
+		<description>This setting has no impact on setpoint temperatures but will make the freezer compartment cool to the
+			lowest possible temperature until it is disabled manually by the customer or by the home appliance because of
+			a
+			timeout.</description>
```

Comment:
Can you fix the line break?

Reply:
`mvn spotless:apply` is creating this

Reply:
If you put the entire tag in one line, spotless should make a better job.